libfuse/mount_test.go

```diff
@@ -1101,6 +1101,40 @@ func TestRemoveDirNotEmpty(t *testing.T) {
 	}
 }
 
+func TestRemoveFileWhileOpenSetEx(t *testing.T) {
```

Comment:
Yeah, looks like you have to do a `futimes` syscall, which is a pain. So nevermind, I guess!